package.json

```
@@ -1,52 +1,56 @@
-{
-  "name": "fin",
-  "version": "1.0.0",
-  "private": true,
-  "description": "helps users track their expenses, manage budgets and receive financial insights directly via Telegram.",
-  "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
-  },
-  "keywords": [
-    "finance",
-    "finance-management",
-    "telegram-mini-app"
-  ],
-  "author": "Mukhammaddiyor Takhirov <opensource@takhirov.uz>",
-  "license": "GPL-3.0-only",
-  "dependencies": {
-    "@telegram-apps/sdk-react": "^2.0.8",
-    "geist": "^1.3.1",
-    "next": "^14.2.15",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
-    "tailwindcss": "^3.4.14",
-    "typescript": "^5.6.3"
-  },
-  "devDependencies": {
-    "@eslint/compat": "^1.2.1",
-    "@eslint/eslintrc": "^3.1.0",
-    "@eslint/js": "^9.13.0",
-    "@next/bundle-analyzer": "^14.2.15",
-    "@next/eslint-plugin-next": "^14.2.15",
-    "@types/node": "^22.7.6",
-    "@types/react": "^18.3.11",
-    "@types/react-dom": "^18.3.1",
-    "@typescript-eslint/eslint-plugin": "^8.10.0",
-    "@typescript-eslint/parser": "^8.10.0",
-    "autoprefixer": "^10.4.20",
-    "clsx": "^2.1.1",
-    "eslint": "^9.12.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-jsx-a11y": "^6.10.0",
-    "eslint-plugin-react": "^7.37.1",
-    "eslint-plugin-react-hooks": "^5.0.0",
-    "eslint-plugin-tailwindcss": "^3.17.5",
-    "globals": "^15.11.0",
-    "postcss": "^8.4.47",
-    "prettier-plugin-tailwindcss": "^0.6.8",
-    "tailwind-merge": "^2.5.4"
-  }
-}
+{
+  "name": "fin",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "description": "helps users track their expenses, manage budgets and receive financial insights directly via Telegram.",
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "prepare": "pnpm switch:env",
+    "switch:env": "node scripts/switchEnv --dev"

```

I noticed you're using `pnpm` for the prepare command, but I would recommend switching to `npm` for better compatibility. Additionally, I suggest renaming the command and script files for clarity.

```suggestion
    "prepare": "npm run generate:env",
    "generate:env": "node scripts/generate_env.mjs"
```